docs/advanced/config_system.md

```diff
@@ -9,6 +9,8 @@ The mbed configuration system customizes the compile time configuration of vario
 
 The configuration system gathers and interprets all the configurations defined in the source tree. The output of the configuration system is a list of macros that are automatically defined when compiling the code.
 
+<span class="notes">**Note:** In prior releases, the configuration system provided a method for adding custom targets. Custom target are handled as [mbed targets](mbed_targets.md) in a special file named `custom_targets.json` in the root of a project.</span>
```

Comment:
"are handled" is passive voice. Could we rewrite this as "This release handles custom targets as [mbed targets](mbed_targets.md) in a special file named `custom-targets.json` in the root of a project." or something similar?